site/docs/use-cases/speech-processing/_sections/_run_model/index.mdx

```diff
@@ -0,0 +1,41 @@
+import CodeExampleCPP from './_code_example_cpp.mdx';
+import CodeExamplePython from './_code_example_python.mdx';
+
+## Run Model Using OpenVINO GenAI
+
+OpenVINO GenAI introduces the [`WhisperPipeline`](https://docs.openvino.ai/2025/api/genai_api/_autosummary/openvino_genai.WhisperPipeline.html) pipeline for inference of speech processing Whisper models.
+You can construct it straight away from the folder with the converted model.
+It will automatically load the model, tokenizer, detokenizer and default generation configuration.
+
+:::info
+`WhisperPipeline` expects normalized audio files in WAV format at sampling rate of 16 kHz as input.
+:::
+
+<LanguageTabs>
+    <TabItemPython>
+        <Tabs groupId="device">
+            <TabItem label="CPU" value="cpu">
```

Comment:
gpu/cpu tabs looks a bit bulky to me. We should also add NPU I believe. What do you think if we replace device tabs with a comment in snippet: `GPU, NPU can be used as well` ?
Should we also add NPU for TextGeneration pipeline?

Reply:
Discussed, will be applied in separate PR